### ISSUE TITLE
NPM out of sync

### ISSUE THREAD

**Issue by fearphage** [tediousjs/tedious#1128]
`npm install tedious@latest` installs `8.3.0` instead of `8.3.1`.

```shell
➜  hq-lb git:(master) npm info --json tedious dist-tags
{
  "latest": "8.3.0",
  "beta": "1.11.5-beta.5-gb7b6e4d",
  "next": "9.0.1"
}
```

PS. The `beta` tag should probably be deleted since it seems completely unmaintained.

**Comment by IanChokS** [comment]
Thanks for raising this issue, we'll take a look on how to keep npm up to date!

**Comment by IanChokS** [comment]
Fixed in #1132
